src/App.tsx

```diff
@@ -1,61 +1,117 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/todo';
+
+const getMaxId = () => {
+  if (todosFromServer.length === 0) {
+    return 1;
+  }
+
+  const maxId = Math.max(...todosFromServer.map(todo => todo.id));
+
+  return maxId + 1;
+};
 
 export const App = () => {
+  const [inputTitle, setInputTitle] = useState('');
+  const [selectUser, setSelectUser] = useState(0);
+  const [hasErrorMessageTitle, setHasErrorMassageTitle] = useState(false);
+  const [hasErrorMessageSelect, setHasErrorMassageSelect] = useState(false);
+  const [todosList, setTodosList] = useState(todosFromServer);
+
+  const handleTitleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
[CODE STYLE] - Don't use one-letter variable naming. Instead of 'e', use 'event' or 'changeEvent'.